lib/Conversion/TorchToTosa/TorchToTosa.cpp

```diff
@@ -3702,28 +3702,67 @@ LogicalResult ConvertAtenOp<AtenArangeStartStepOp>::matchAndRewrite(
         op, "unimplemented: pin_memory must be either None or false");
   }
 
-  int64_t start, step, end;
-  if (!matchPattern(op.getStart(), m_TorchConstantInt(&start)))
+  auto matchIntOrDouble =
+      [&](Value val) -> std::tuple<LogicalResult, int64_t, double> {
+    // Match int or fp values. The one used depends on the resultType.
+    // Therefore `valueInt` and `valueDouble` will have similar values (but may
+    // be truncated due to casting).
```

Comment:
Could you update the comment to say how they reflect conversions. Are all potential type conversions accounted for, or are some on `TODO`?

Reply:
All potential conversions are accounted for. I don't think there is any type conversion missing.